view/frontend/templates/subscribe.phtml

```diff
@@ -0,0 +1,9 @@
+<?php
+//Don't care if form collection is disabled as bots would get added also with sync only.
```

Comment:
Formatting: missing space between `//` and `Don't`.

Reply:
Fixed in 5b01525b67a526cd7dd2ee22e2451f31d2d632ce